goss.yaml

```diff
@@ -1,9 +1,6 @@
 process:
   mimir:
     running: true
-package:
-  ca-certificates:
-    installed: true
```

Comment:
And now, given the new context, I agree with Dylan that we can keep this "rock linting" check in place.
We would remove the `package` section if we were to convert this from "internal" to "external" check.

Reply:
Got you, keeping that check then!